src/Config.ts

```diff
@@ -22,6 +22,44 @@ export class SocketOptions {
     //TO-DO
 }
 
+export interface SslOptions {
```

Comment:
I reconsidered it and I think interface is ok. 

Just last and a very minor point: Can we make name of the class `SSLOptions` since SSL is an abreviation?